SherpaBackEnd/Analysis/Domain/HackmanAnalysis.cs

```diff
@@ -1,35 +1,52 @@
+using MongoDB.Driver.Linq;
+
 namespace SherpaBackEnd.Analysis.Domain;
 
 public class HackmanAnalysis
 {
-    public HashSet<string> Categories { get; }
-    
-    public List<SurveyResult<string>> Surveys = new();
+    public List<string> Categories => GetCategories();
+
+    private readonly HashSet<string> _categories = new();
+
+    public readonly List<SurveyResult<string>> Surveys = new();
 
     public double Average => GetAverage();
 
     public readonly double Aspirational = 0.75;
 
     public HackmanAnalysis(List<SurveyAnalysisData<string>> surveyAnalysisDatas)
     {
-        Categories = new HashSet<string>();
         foreach (var survey in surveyAnalysisDatas)
```

Comment:
could we name this surveyAnalysisData instead of Datas? I believe in English it's an uncountable word